plugins/org.polarsys.capella.scenario.editor.dsl/src-gen/org/polarsys/capella/scenario/editor/dsl/textualScenario/Role.java

```diff
@@ -8,7 +8,7 @@
  *  SPDX-License-Identifier: EPL-2.0
  *  
  *  Contributors:
- *     Thales - initial API and implementation
+ *       Thales - initial API and implementation
  ******************************************************************************/
 /**

```

Comment:
I think if the header you applied had 2 " " removed, this won't show diff on so many files.

Reply:
Done.